app/controllers/stripe-setup/add-psp-account-details/get-controller.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+const paths = require('../../../paths')
+const logger = require('../../../utils/logger')(__filename)
+const { response, renderErrorView } = require('../../../utils/response')
+const { ConnectorClient } = require('../../../services/clients/connector_client')
+const connectorClient = new ConnectorClient(process.env.CONNECTOR_URL)
+
+module.exports = async (req, res) => {
+  try {
+    const stripeAccountSetup = await connectorClient.getStripeAccountSetup(
+      req.account.gateway_account_id, req.correlationId)
```

Comment:
_Very minor alert_, if the method call goes over a line it's useful to have it consistent just when skimming through code and looking for the other bracket, something like:
```js
const stripeAccountSetup = await connectorClient.getStripeAccountSetup(
      req.account.gateway_account_id, req.correlationId
)
```

or (potentially preferable here): 
```js
const stripeAccountSetup = await connectorClient.getStripeAccountSetup(
    req.account.gateway_account_id,
    req.correlationId
)
```

Reply:
cool will change this in the next PR to save having to wait for another build